repository/src/main/java/org/apache/atlas/repository/graph/GraphHelper.java

```diff
@@ -1025,6 +1025,20 @@ public static String getGuid(AtlasVertex vertex) {
         return ret;
     }
 
+    public static String getQalifiedName(AtlasVertex vertex) {
+        return vertex.<String>getProperty(Constants.QUALIFIED_NAME, String.class);
+    }
+
+    public static Map<String,String> getNodeDetails(AtlasVertex vertex) {
```

Comment:
can rename to fetchAttributes(vertex, attributesList)
Maintian attributesList as a constant List which can be modified later.